### PR TITLE
Docs: update zh/cache_management.md

### PR DESCRIPTION
**Pull request by yuhr123**

### CODE REVIEW

**Comment by davies** [comment]
@yuhr123 Please update the English docs also.

**Comment by yuhr123** [comment]
> @yuhr123 Please update the English docs also.

done.

File: docs/en/administration/cache_management.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ slug: /cache_management
 
 For a file system driven by a combination of object storage and database, the cache is an important medium for efficient interaction between the local client and the remote service. Read and write data can be loaded into the cache in advance or asynchronously, and then the client interacts with the remote service in the background to perform asynchronous uploads or prefetching of data. The use of caching technology can significantly reduce the latency of storage operations and increase data throughput compared to interacting with remote services directly.
 
+## JuiceFS Read Cache mechanism
+
+When reading files in JuiceFS, there are multiple levels of caches to provide better performance for frequently accessed data. Read requests will try the in-memory page cache, the pre-read buffer of the JuiceFS process, and the local disk cache in turn, and will read from the object storage only when the corresponding data is not found in the cache, and will write to all levels of caches asynchronously to ensure the performance of the next access.
```
Comment (SandyXSD) [inline docs/en/administration/cache_management.md:12]: "in-memory page cache" -> "kernel page cache"

File: docs/zh_cn/administration/cache_management.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ slug: /cache_management
 
 对于一个由对象存储和数据库组合驱动的文件系统，缓存是本地客户端与远端服务之间高效交互的重要纽带。读写的数据可以提前或者异步载入缓存，再由客户端在后台与远端服务交互执行异步上传或预取数据。相比直接与远端服务交互，采用缓存技术可以大大降低存储操作的延时并提高数据吞吐量。
 
+## JuiceFS 的读缓存机制
+
+当访问 JuiceFS 中的文件时，会有多级缓存给经常访问的数据提供更好地性能，读请求会依次尝试内存分页缓存、JuiceFS进程的预读缓冲区、本地磁盘缓存，当缓存中没找到对应数据时才会从对象存储读取，并且会异步写入各级缓存保证下一次访问的性能。
```
Comment (SandyXSD) [inline docs/zh_cn/administration/cache_management.md:12]: 内存 -->  内核

**Review by SandyXSD — APPROVED** [review]

**Pull request merged**